frontend/src/server/APIrequests.ts

```diff
@@ -0,0 +1,27 @@
+import { IArticles } from '../types/commonTypes';
```

Comment:
would like to create some @tc/common-types or @tc/types to create non relative imports